sqlite3/lib/src/wasm/file_system.dart

```diff
@@ -35,6 +35,9 @@ abstract class FileSystem {
   /// otherwise.
   void deleteFile(String path);
 
+  /// List all files
+  List<String> listFiles();
```

Comment:
Let's expand this a bit:

```suggestion
  /// List all files.
  ///
  /// Returns a list of paths for files that currently exist in this file system.
  List<String> listFiles();
```